doc/user/content/sql/alter-cluster.md

```diff
@@ -6,44 +6,159 @@ menu:
     parent: 'commands'
 ---
 
-`ALTER CLUSTER` changes the configuration of a cluster, which **might incur
-downtime**. To rename a cluster, use [`ALTER ... RENAME`](/sql/alter-rename/).
+`ALTER CLUSTER` changes the configuration of a cluster, such as the `SIZE` or
+`REPLICATON FACTOR`.
 
 ## Syntax
 
-#### Set new configuration value
+`ALTER CLUSTER` has the following syntax forms:
 
-{{< diagram "alter-cluster-set.svg" >}}
+{{< tabs >}}
+{{< tab "Set a configuration" >}}
```

Comment:
```suggestion
{{< tab "Change configuration values" >}}
```

Reply:
Oh ... some of this repetition of the exact command syntax is for search results purposes.  But, absolutely understand your intent though.